### PR TITLE
Task 2.1 done

### PR DESCRIPTION
**Pull request by pchizhov**
В ходе работы над задачей была написана простая нейронная сеть из двух слоев.

Точность на тренировочных данных составила 0.999
Точность на валидационных данных составила 0.773
Точность на тестовых данных составила 0.739

### CODE REVIEW

**Comment by pchizhov** [comment]
Reviewers: @toshiks

**Comment by pchizhov** [comment]
Сорри, в первый раз не сохранились изменения с убранными выводами в notebook-e

**Comment by pchizhov** [comment]
Задание 2.2 сделано.

В результате работы была натренирована многослойная нейросеть, реализованная с помощью фреймворка PyTorch. Точность на валидационных данных составила 0.82, точность на тестовых данных составила 0.79. Наилучшего результата удалось добиться, используя дополнительный линейный слой, с количеством нейронов в слое 100, learning rate 0.001 и weight_decay 0.00001.
